Source/Scene/Points3DTileContent.js

```diff
@@ -148,19 +155,42 @@ define([
         }
     });
 
+    function createFeatures(content) {
```

Comment:
I think I have a better solution built off the discussion above:

If the BATCH_ID semantic is present, then the points are considered batched. This is the right technique for differentiating physical features within the point cloud, like doors vs windows vs walls. A `Cesium3DTileBatchTable` is used to individually select, color, and pick sections, as well as get/set batch properties.

If the BATCH_ID is not present, then the batch table stores per-point properties. In order to avoid the memory overhead of creating a `Cesium3DTileFeature`/color/show/pickID for every point, a `Cesium3DTileBatchTable` is not created. Instead the properties are inputted to the shader as vertex attributes (much easier to work with than textures and all styling happens in the vertex shader anyways) and styled in the shader. The properties cannot be get/set.

So overall the concept is, if BATCH_IDs are defined for the tile, then the point cloud is "batched" and uses a batch table like normal. Otherwise there is no batch table, and per-point properties are only available to the shader for styling.